core/templates/pages/donate-page/donate-page.module.ts

```diff
@@ -20,16 +20,13 @@ import 'core-js/es7/reflect';
 import 'zone.js';
 
 angular.module('oppia', [
-  'dndLists', 'headroom', 'infinite-scroll', 'ngAnimate',
-  'ngAudio', require('angular-cookies'), 'ngImgCrop', 'ngJoyRide', 'ngMaterial',
-  'ngSanitize', 'ngTouch', 'pascalprecht.translate',
-  'toastr', 'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate',
+  'headroom', require('angular-cookies'), 'ngTouch', 'ngSanitize',
+  'pascalprecht.translate', 'toastr', 'ui.bootstrap'
 ]);
 
 import { Component, NgModule, StaticProvider } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { downgradeComponent } from '@angular/upgrade/static';
-import { FormsModule } from '@angular/forms';
```

Comment:
Btw, do we have something like unused imports check for TypeScript? cc @kevinlee12

Reply:
/cc @Hudda

Reply:
It's in the master https://github.com/oppia/oppia/issues/8423 linter list as unused var, I've pinged the contributor that was assigned this.